wp-content/civi-extensions/goonjcustom/goonjcustom.php

```diff
@@ -145,23 +149,53 @@ function goonjcustom_evaluate_tokens(TokenValueEvent $e) {
 			$inductionActivity = $inductionActivities->first();
 			$inductionDateTime = $inductionActivity['activity_date_time'] ?? 'Not Scheduled';
 			$inductionGoonjOffice = $inductionActivity['Induction_Fields.Goonj_Office'] ?? '';
+      $inductionAssigneeId = $inductionActivity['Induction_Fields.Assign']?? '';
 
 			// Fetch office online meet link details if induction office is specified
 			$inductionOnlineMeetlink = '';
+      $completeAddress = '';
+      $inductionOfficeCity = '';
+      $inductionAssigneeFirstName = '';
+      $InductionAssigneePhone = '';
+
+      if( $inductionAssigneeId){
+        $assigneeDetails = \Civi\Api4\Contact::get(FALSE)
+          ->addSelect('first_name', 'phone.phone')
+          ->addJoin('Phone AS phone', 'LEFT')
+          ->addWhere('id', '=', $inductionAssigneeId)
+          ->execute()->single();
+
+        $inductionAssigneeFirstName = $assigneeDetails['first_name'];
+        $InductionAssigneePhone = $assigneeDetails['phone.phone'];
+
+      }
+
 			if ($inductionGoonjOffice) {
 				$officeDetails = \Civi\Api4\Contact::get(FALSE)
-					->addSelect('Goonj_Office_Details.Induction_Meeting_Access_Link')
+					->addSelect('Goonj_Office_Details.Induction_Meeting_Access_Link', 'address_primary.city')
 					->addWhere('contact_sub_type', 'CONTAINS', 'Goonj_Office')
 					->addWhere('id', '=', $inductionGoonjOffice)
 					->execute()->single();
 
 				$inductionOnlineMeetlink = $officeDetails['Goonj_Office_Details.Induction_Meeting_Access_Link'] ?? '';
+        $inductionOfficeCity = $officeDetails['address_primary.city'] ?? '';
+        $addresses = \Civi\Api4\Address::get( false )
+          ->addWhere( 'contact_id', '=', $officeDetails['id'] )
+          ->addWhere( 'is_primary', '=', true )
+          ->setLimit( 1 )
+          ->execute();
+
+        $address = $addresses->count() > 0 ? $addresses->first() : null;
+        $completeAddress = CRM_Utils_Address::format($address);
 			}
 
 			// Assign tokens
 			$row->tokens('contact', 'inductionDateTime', $inductionDateTime);
 			$row->tokens('contact', 'inductionOnlineMeetlink', $inductionOnlineMeetlink);
-
+      $row->tokens('contact', 'InductionOfficeAddress', $completeAddress);
+      $row->tokens('contact', 'InductionOfficeCity', $inductionOfficeCity);
+      $row->tokens('contact', 'InductionAssigneeFirstName', $inductionAssigneeFirstName);
+      $row->tokens('contact', 'InductionAssigneePhone', $InductionAssigneePhone);
```

Comment:
Change `InductionAssigneeFirstName` to `inductionAssigneeFirstName` using camel case. Apply the same camel case format to other similar variables as well.